scheduleapp/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+# Create your models here.
+class Activity(models.Model):
+
+    content = models.CharField(max_length=500)
+    week_day = models.CharField(max_length=25)
```

Comment:
Please add choices for days: https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.Field.choices